File: javacommons/fileutils/tsrc/driveutilities/src/alltests.cpp
/*
* Copyright (c) 2008 Nokia Corporation and/or its subsidiary(-ies).
* All rights reserved.
* This component and the accompanying materials are made available
* under the terms of "Eclipse Public License v1.0"
* which accompanies this distribution, and is available
* at the URL "http://www.eclipse.org/legal/epl-v10.html".
*
* Initial Contributors:
* Nokia Corporation - initial contribution.
*
* Contributors:
*
* Description:  AllTests
*
*/


#include <stdio.h>

#include "commsclientendpoint.h"

#include "Platform.h"
#include "CommandLineTestRunner.h"

using java::comms::CommsClientEndpoint;

int main(int ac, char** av)
{
    // kludge to get rid of mem leak warnings due to global context impl in comms
    {
        CommsClientEndpoint client2;
    }

    int status = CommandLineTestRunner::RunAllTests(ac, av);

    getchar();

    return status;
}

#include "alltests.h"

File: javacommons/jvms/cldc_1.1.1/tsrc/javasrc/com/nokia/mj/test/java/util/AllTests.java
/*
* Copyright (c) 2008 Nokia Corporation and/or its subsidiary(-ies).
* All rights reserved.
* This component and the accompanying materials are made available
* under the terms of "Eclipse Public License v1.0"
* which accompanies this distribution, and is available
* at the URL "http://www.eclipse.org/legal/epl-v10.html".
*
* Initial Contributors:
* Nokia Corporation - initial contribution.
*
* Contributors:
*
* Description:
*
*/


package com.nokia.mj.test.java.util;

import com.nokia.mj.impl.installer.utils.InstallerMain;

import j2meunit.framework.TestCase;
import j2meunit.framework.TestMethod;
import j2meunit.framework.TestSuite;

public class AllTests extends TestCase implements InstallerMain
{
    public void installerMain(String[] args)
    {
        TestSuite suite = new TestSuite(this.getClass().getName());

        suite.addTest(new TimerTaskTest().suite());
        suite.addTest(new TimerTest().suite());

        com.nokia.mj.impl.utils.OmjTestRunner.run(suite);
    }

    public AllTests() {}
    public AllTests(String aTestName, TestMethod aTestMethod)
    {
        super(aTestName, aTestMethod);
    }

    protected void setUp() {}
    protected void tearDown() {}
}
